### PR TITLE
bug: NoUnusedImportsFixer - fix for splat operator

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6829

### CODE REVIEW

File: tests/Fixer/Import/NoUnusedImportsFixerTest.php [hunk 1]
```diff
@@ -1515,5 +1515,16 @@ enum Foo: string
                 }
             ',
         ];
+
+        yield [
```
Comment (keradus) [inline tests/Fixer/Import/NoUnusedImportsFixerTest.php:1519]: can we add example without `...` as well?

File: src/Tokenizer/AbstractTypeTransformer.php [hunk 2]
```diff
@@ -56,13 +56,19 @@ private function isPartOfType(Tokens $tokens, int $index): bool
...
+        }
+
+        if (!$tokens[$afterTypeIndex]->isGivenKind(T_VARIABLE)) {
             return false;
         }
-        $beforeVariableIndex = $tokens->getPrevMeaningfulToken($variableIndex);
-        if ($tokens[$beforeVariableIndex]->equals('&')) {
+
+        $beforeVariableIndex = $tokens->getPrevMeaningfulToken($afterTypeIndex);
+        if ($tokens[$beforeVariableIndex]->equalsAny(['&'])) {
```
Comment (keradus) [inline src/Tokenizer/AbstractTypeTransformer.php:71]: why `equalsAny` instead of `equals` ?

Comment (kubawerlos) [reply src/Tokenizer/AbstractTypeTransformer.php]: I was trying `[T_ELLIPSIS]` there and forgot to revert.

Comment (Wirone) [reply src/Tokenizer/AbstractTypeTransformer.php]: Looks like good candidate for fixer 😝

**Pull request merged**